service/processor/src/main/java/com/google/auto/service/processor/ServicesFiles.java

```diff
@@ -34,6 +34,8 @@
 final class ServicesFiles {
   public static final String SERVICES_PATH = "META-INF/services";
 
+  public static final char NEWLINE = '\n';
```

Comment:
I'm not sure it's worth defining a constant for this. Maybe just do `writer.write('\n')` below?

Reply:
will change :)